apps/web/modules/settings/my-account/general-view.tsx

```diff
@@ -202,6 +210,7 @@ const GeneralView = ({ localeProp, user, travelSchedules, revalidatePage }: Gene
             locale: values.locale.value,
             timeFormat: values.timeFormat.value,
             weekStart: values.weekStart.value,
+            defaultView: values.defaultView.value as "EVENT_TYPES" | "BOOKINGS" | "INSIGHTS",
```

Comment:
Instead of using these constant strings, we can have an enum and use it; that will be better.